### ISSUE TITLE
Test batch size

### ISSUE THREAD

**Issue by Youssef-s** [Woodman718/CapsNets#2]
I am really interested in your work, but I have a question that I couldn't understand in the code.
How does the resulting accuracy change when you change the test batch size ?
I can't find any parameter or operation done in testing that depends on batch size.

**Comment by Woodman718** [comment]
Thank you for your interest in my work. 
It remains stable and performs optimally under the current model and overparameter settings. 
Regarding the effects of hyperparameters on the model, such as batch size, seed, and learning rate, you can find a detailed discussion in Chapter 4, Section 4 of my graduation thesis titled 'Research on Skin Cancer Assisted Diagnosis Method Based on FixCaps.' 
However, I would like to emphasize that these parameters are not the primary focus of the study. 
In this paper（https://doi.org/10.36227/techrxiv.23291003）, we introduce matrix norm and condition number into the loss function, and we also uncover the unique characteristics of the convolution layer with a kernel size of 21.  
These findings may offer more intriguing research avenues.
